src/libraries/System.Private.CoreLib/docs/Unordered Vectorized Dictionary Proposal.md

```diff
@@ -0,0 +1,316 @@
+# Unordered Vectorized Dictionary
```

Comment:
We have our API proposal process documented at https://github.com/dotnet/runtime/blob/main/docs/project/api-review-process.md . New API proposals should start with an issue.

Committing docs with API proposals to main is not following the established process, and it is going to cause confusion.